webxrlayers-1.bs

```diff
@@ -439,8 +441,19 @@ If the layer was created without depth/stencil, this attribute returns null.
 The <dfn attribute for="XRWebGLSubImage">imageIndex</dfn> attribute returns the offset into the texture array. Valid only for layers
 that were requested with {{texture-array}}.
 
-This {{XRWebGLSubImage/colorTexture}} and {{XRWebGLSubImage/depthStencilTexture}} are only valid during
-{{XRSession/requestAnimationFrame()}} callback and becomes invalid as soon as the callback returns.
+The {{XRWebGLSubImage/colorTexture}} and {{XRWebGLSubImage/depthStencilTexture}} objects MUST only be used during the
+{{XRSession/requestAnimationFrame()}} callback of the current session and MUST be invalid as soon as the callback returns.
```

Comment:
Not an issue introduced by this PR, but on reading this I realized this should actually say "MUST only be used during an [=XR Animation frame=] of the current session and MUST be made invalid once the [=XR Animation frame=] completes."

The reason for this is that an animation frame may be comprised of multiple individual rAF callbacks, and we actually don't want to invalidate the texture handle after every once since the texture will only be sent to the compositor once all of them complete.

Reply:
I replaced the text with your suggestion.